kentikapi/credentials.go

```diff
@@ -24,10 +19,11 @@ const DefaultProfile = "default"
 // 5. file at ./.kentik/default
 
 // GetCredentials reads and returns (email, token) pair from environment variables or specified profile config file.
-func GetCredentials(profile string) (authEmail, authToken string, err error) {
+// Default profileName name can be specified by global variable DefaultProfile.
```

Comment:
```suggestion
// Profile name specifies file name containing Kentik profile data. DefaultProfile constant can be used for that.
```
It's not variable, it's a const.

Reply:
Done